detections/endpoint/RMM_Agent_Install_Followed_By_First_Outbound_Connection.yml

```diff
@@ -0,0 +1,117 @@
+name: RMM Agent Install Followed By First Outbound Connection
+id: null
+version: 1
+date: '2025-08-17'
+author: Riyan Pratama (sudo3rs)
+data_source:
+  - Sysmon
+  - Windows Security Event Log
+  - Windows System Event Log
+  - Web Proxy
+  - Firewall
+type: TTP
+status: production
+description: |
+  Detects installation or service registration of common Remote Monitoring & Management (RMM) tools
+  (e.g., AnyDesk, ConnectWise ScreenConnect, Atera, Splashtop, TeamViewer) followed within a short
+  window by the first outbound connection from the same host to the vendor’s cloud endpoints.
+  This correlation highlights “living-off-RMM” abuse where an attacker deploys a legitimate RMM
+  agent for persistent remote access, then immediately dials out to external control infrastructure.
+  A SOC should investigate promptly, as this often indicates hands-on-keyboard activity enabling
+  remote control, data staging/exfiltration, or further lateral movement.
+search: |
+  (
+    (`wineventlog_system` EventCode=7045) OR
+    (`wineventlog_security` EventCode=4697) OR
+    (`sysmon` EventCode=1)
+  )
+  (Image="*anydesk*.exe" OR Image="*screenconnect*" OR Image="*connectwise*" OR Image="*atera*" OR Image="*splashtop*" OR Image="*teamviewer*"
+   OR ServiceFileName="*AnyDesk*" OR ServiceFileName="*ScreenConnect*" OR ServiceFileName="*ConnectWise*" OR ServiceFileName="*Atera*" OR ServiceFileName="*Splashtop*" OR ServiceFileName="*TeamViewer*")
```

Comment:
We already have a lookup with binary names and domains. I suggest you use that as it can easily be updated instead https://github.com/splunk/security_content/blob/develop/lookups/remote_access_software.csv